### ISSUE TITLE
Unclear if NPM installing an EVM package works differently than zos link.

### ISSUE THREAD

**Issue by crazyrabbitLTC** [OpenZeppelin/openzeppelin-sdk#781]
In the testing guide: https://docs.zeppelinos.org/docs/testing.html
We start our project with: "npm install zos zos-lib openzeppelin-eth truffle chai"
This installs openzeppelin-eth (as well as the other packages). 
However later in the testing guide we do: "npx zos link openzeppelin-eth". 
The feedback from this command is: "Installing openzeppelin-eth from npm...".

My question is: does zos link check if the package is already installed? Why do we bother with npm install openzeppelin-eth earlier if zos link simply re-installs from npm?  Is there a difference between the two installation methods and a reason behind this? Perhaps this is just a typo?

**Comment by jcarpanelli** [comment]
Regarding your questions:
1)  **Does zos link check if the package is already installed?** 
Nope, but we should.
2) **Why do we bother with npm install openzeppelin-eth earlier if zos link simply re-installs from npm?**
I believe we can remove openzeppelin-eth from the list as it's going to be installed through `zos link`
3) **Is there a difference between the two installation methods and a reason behind this?** 
When running `zos link`, `zos` not only `npm install`s the package but also adds it into the `zos.json` as a linked dependency.

In conclusion: we can remove `openzeppelin-eth` from the `npm install` list.

**Comment by spalladino** [comment]
Closing in favour of #1317
